src/services/GithubService.js

```diff
@@ -0,0 +1,16 @@
+import Axios from "axios";
+
+export default {
+    getPullRequests: (username) => {
+        const query = "type:pr author:" + username + " -user:" + username;
+        const url = "https://api.github.com/search/issues";
+
+        return Axios.get(url, {
+            params: { q: query }
+        });
+    },
+    getPullRequestInfo: (issue) => {
```

Comment:
@lazyc97 May be it would be more informative if we will rename `getPullRequestInfo` to `getPullRequestInfoByIssue`. What do you think?
There's the last one comment :)